rest/route/service.go

```diff
@@ -103,7 +103,8 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/task/{task_id}/set_results_info").Version(2).Post().Wrap(requireTask).RouteHandler(makeSetTaskResultsInfoHandler())
 	app.AddRoute("/task/{task_id}/start").Version(2).Post().Wrap(requireTask, requirePodOrHost).RouteHandler(makeStartTask(env))
 	app.AddRoute("/task/{task_id}/test_logs").Version(2).Post().Wrap(requireTask, requirePodOrHost).RouteHandler(makeAttachTestLog(settings))
-	app.AddRoute("/task/{task_id}/git/patch").Version(2).Get().Wrap(requireTask).RouteHandler(makeGitServePatch())
+	app.AddRoute("/task/{task_id/patch").Version(2).Get().Wrap(requireTask).RouteHandler(makeServePatch())
```

Comment:
We have to maintain temporary backward compatibility for agents that are collecting task data but are still on the old agent version because those old agents still request the patch from `/task/{task_id}/git/patch`. An easy way to do this is to temporarily keep the `/task/{task_id}/git/patch` route while still introducing the new `/task/{task_id}/patch` route. Then once this is deployed and the agents have rolled over, remove `/task/{task_id}/git/patch` in a follow-up PR.

Reply:
That sounds good to me!